recipes/python-constructs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "constructs" %}
+{% set version = "10.1.158" %}
+
+package:
+  name: python-{{ name|lower }}
```

Comment:
Is there a non-python constructs package wrapper? If not, then the package name should just be 'constructs' to match the PYPI name.

Reply:
The constructs repo is a JavaScript project with packages for several different languages including Python https://github.com/aws/constructs
Since the main purpose of this is to allow the [aws-cdk](https://github.com/conda-forge/staged-recipes/issues/16166) to be packaged which is also multi-language I think it makes sense to keep the python prefix.